extensions/espanso/src/lib/espanso.ts

```diff
@@ -0,0 +1,29 @@
+import { getPreferenceValues } from "@raycast/api";
+import { arch } from "os";
+import { exec } from "child_process";
+
+function execAsync(command: string): Promise<{ stdout: string; stderr: string }> {
+  return new Promise((resolve, reject) => {
+    exec(command, (error, stdout, stderr) => {
+      if (error) {
+        reject(error);
+        return;
+      }
+      resolve({ stdout, stderr });
+    });
+  });
+}
+
+export async function espansoCli(args: string) {
+  const preferences = getPreferenceValues<Preferences>();
+
+  const espansoPath: string = preferences.espansoPath?.length
+    ? preferences.espansoPath
+    : arch() === "arm64"
+      ? "/opt/homebrew/bin/espanso"
+      : "/usr/local/bin/espanso";
```

Comment:
I wonder if we could test first if it exists because not everyone installs it via homebrew.

Wonder if we could check first if the shortcut command `espanso` exists and once it does not exist, test first the preference and then the homebrew version.